picard/browser/browser.py

```diff
@@ -65,11 +65,19 @@ def stop(self):
 
     def _process_request(self):
         conn = self.sender()
-        line = bytes(conn.readLine()).decode()
-        conn.write(b"HTTP/1.1 200 OK\r\nCache-Control: max-age=0\r\n\r\nNothing to see here.")
-        conn.disconnectFromHost()
+        rawline = conn.readLine().data()
+        log.debug("Browser integration request: %r", rawline)
+        try:
+            line = rawline.decode()
+            conn.write(b"HTTP/1.1 200 OK\r\nCache-Control: max-age=0\r\n\r\nNothing to see here.")
```

Comment:
Not really a big fan of hardcoded strings like these. Move it to constants maybe?